packages/backend/server/routes/[packageAndRefOrSha].get.ts

```diff
@@ -0,0 +1,42 @@
+import { WorkflowData } from "../types";
+
+type Params = Omit<WorkflowData, "sha" | "isPullRequest" | "ref"> & {
+  packageAndRefOrSha: string;
+};
+
+interface PackageData {
+  repository?: {

```

Comment:
hmm, is there any edge cases that we need to handle? what does the npm doc says about this? 

just asking, just if the repository might be string or anything else. But we should see what the doc says.